db/migrate/20151103223107_drop_properties.rb

```diff
@@ -0,0 +1,5 @@
+class DropProperties < ActiveRecord::Migration
+  def change
```

Comment:
Yeah understood that we don't want to restore the table. I guess the reason I originally suggested marking is irreversible is to reveal intent. See: http://maxwellholder.com/blog/rails-migration-checklist

I can see that there's no precedent for doing that, but do you think we should start? In most cases, reversible migrations are the way to go. This is obviously an exception so perhaps we should indicate it as such?